content/ko/docs/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Docs Home"
+description: "Everything you need to know about how the documentation is organized."
+---
+
+Translate this doc into Chinese, please!
```

Comment:
this doc into Chinese --> this doc into Korean ??

Reply:
oops. thanks for catching that. You also really noticed my copy/paste strategy to this PR.